dev/examples/controls/bar.js

```diff
@@ -415,18 +414,16 @@ export const renderBarWithPanning = (id) => {
             }
         ]
     };
-    const createGraph = (axis, values) => {
-        if (graph) {
-            graph.destroy();
-        }
-        graph = Carbon.api.graph(axis);
-        graph.loadContent(Carbon.api.bar(values));
-        return graph;
+    const createGraph = () => {
+        graph.reflow();
```

Comment:
How does this work? Where is `graph` defined?

Reply:
It is defined over [here](https://github.com/cerner/carbon-graphs/blob/3950e76abce87cfe4e00372d5f86053477ae5c11/dev/examples/controls/bar.js#L421)